packages-engine/core/src/types.ts

```diff
@@ -15,7 +15,7 @@ export type PartialByKeys<T, K extends keyof T = keyof T> = FlatObjectTuple<Part
 export type RequiredByKey<T, K extends keyof T = keyof T> = FlatObjectTuple<Required<Pick<T, Extract<keyof T, K>>> & Omit<T, K>>
 
 export type CSSObject = Record<string, string | number | undefined>
-export type CSSEntry = [string, string | number | undefined]
+export type CSSEntry = [string, string | number | undefined, '$$mini-no-negative'?]
```

Comment:
I got your means. Cause the `negative` variant will process the css entry all of utility, considering the future scalability of the entry, have you optimized the design?

Reply:
Currently, the second parameter is used as the basis for judgment. Considering the scalability, the last parameter can be used to judge whether it is `$$mini-no-negative`, or do you have any better suggestions?